### PR TITLE
Error pages. #46

### PR DESCRIPTION
**Pull request by skrishnateja**
Simple error pages with error code and description are shown.

### CODE REVIEW

File: app/views/errors/error.html.haml [hunk 1]
```diff
@@ -0,0 +1 @@
+%h1.text-center= @error_code+' - '+@error
```
Comment (WaKeMaTTa) [inline app/views/errors/error.html.haml]: Always add an empty line in the last of the line. Is better for `git`

**Comment by skrishnateja** [comment]
These error pages are loaded from Rails application code. Ruby bench is using cowboy, not exactly sure, how cowboy handles error requests.

**Comment by skrishnateja** [comment]
Please take a look @PTAL @tgxworld @kirs

File: app/controllers/errors_controller.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class ErrorsController < ApplicationController
+	def show
+	  errors = { '400' => 'Bad Request', '404' => 'Not Found', '403' => 'Forbidden', '422' => 'Unprocessable Entity',
+	             '502' => 'Bad Gateway', '503' => 'Service Unavailable', '500' => 'internal error' }
+	  respond_to do |format|
+	  	@error_code = params[:error_code] || '999'
+	  	@error = errors[@error_code] || 'Something went wrong'
```
Comment (tgxworld) [inline app/controllers/errors_controller.rb]: The indentation is off here.

Comment (WaKeMaTTa) [inline app/controllers/errors_controller.rb]: Use `spaces` insted of `tabs`. Thanks.

**Comment by kirs** [comment]
Is there any reason why we want to use `exceptions_app` and render the error with controller? I think static `public/404.html` works just fine.

**Comment by skrishnateja** [comment]
Hi @kirs. I am a GSOC enthu, currently I am doing my Internship at Dell, PSR Testing (Performance Scalability and Reliability) Team, I have come across this project in GSOC. I am interested in applying for Ruby Bench. To refresh my Ruby on Rails skills, thought of fixing an issue so, selected the easiest issue #46. This is a minimal contribution. Other than server internal errors, these error pages works fine. Nginx or Cowboy can be mapped to static error pages during maintenance or critical situations. You can close the issue #46 raised 2 years ago and has a help wanted tag.

**Comment by kirs** [comment]
Great to see new GSOC students working on the project!

My impression is that https://github.com/ruby-bench/ruby-bench-web/issues/46 is about styling `404.html`, not moving it into the controller side. Moving errors to be rendered on the controller level us the last measure which can be justified when we want the error page to be in the app's layout. I think that for Rubybench we can stay with a static error page. Basecamp [error page](https://3.basecamp.com/30769866666) can be a good example.

**Comment by skrishnateja** [comment]
Fine. I will try to do the way you mentioned. But I am very bad at writing CSS, Closing this PR as of now.

**Pull request closed without merging**